.github/workflows/scripts/check_diff.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Parse command line arguments
+head_ref=${1}
+base_ref=${2}
+clone_url=${3}
+
+# Add forked repository as remote
+# Fetch base branch from main repository
+# Fetch head branch from forked repository
+git remote add fork ${clone_url}
+git fetch origin ${base_ref}
+git fetch fork ${head_ref}
+
+# Save output of git diff to inspect files changed
+git diff --name-only --diff-filter=ACMRTUXB origin/${base_ref}..fork/${head_ref} > check_diff.txt
+echo "Files changed:"
+cat check_diff.txt
+
+# Set paths to ignore (please test grep command below when adding new patterns)
+paths_ignore=()
+paths_ignore+=("Docs/")
+paths_ignore+=(".github/")
```

Comment:
This
```bash
paths_ignore+=(".github/")
```
was added for testing purposes and should be removed before the PR is merged, if approved.